libarchive/test/test_read_format_rar5.c

```diff
@@ -27,7 +27,6 @@
 /* Some tests will want to calculate some CRC32's, and this header can
  * help. */
 #define __LIBARCHIVE_BUILD
-#include <archive_crc32.h>
 #include <archive_endian.h>
```

Comment:
Thank you for removing this dependency on `archive_crc32.h` (which is private to libarchive).   As you say, we should also remove the dependency on `archive_endian.h` someday, but that can be a different PR.